src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -52,11 +52,26 @@ file that was distributed with this source code.
             </div>
             {% if results_count > 0 %}
                 <div class="box-body no-padding">
-                    <ul class="nav nav-stacked sonata-search-result-list">
+                    <ul class="nav nav-stacked sonata-search-result-list products-list">
```

Comment:
Why `products`?

Reply:
Because this is the most proper selector I found at `AdminLTE.css` for this case:
https://github.com/sonata-project/SonataAdminBundle/blob/710434264761947b468a5c842f90f55cb8a8a90c/src/Resources/public/vendor/admin-lte/dist/css/AdminLTE.css#L2609

Reply:
Does this have any effect on the styling? Can you provide a screenshot without that class? 

The *old* style looks the same to me.

Reply:
Sure. Here you are:
![image](https://user-images.githubusercontent.com/1231441/114240269-a1c54380-995d-11eb-8d6c-0bccfc4e04de.png)